CarpetLib/src/restrict_4d_rf2.cc

```diff
@@ -101,13 +101,13 @@ void restrict_4d_rf2(T const *restrict const src,
   ptrdiff_t const dstkoff = dstoff[2];
   ptrdiff_t const dstloff = dstoff[3];
 
-// Loop over coarse region
-#pragma omp parallel for collapse(4)
+  // Loop over coarse region
+  // #pragma omp parallel for collapse(4)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.